images/oc-build-deploy-dind/build-deploy.sh

```diff
@@ -40,6 +40,10 @@ DOCKER_REGISTRY_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
 
 docker login -u=jenkins -p="${DOCKER_REGISTRY_TOKEN}" ${OPENSHIFT_REGISTRY}
 
+if [ ! -z ${INTERNAL_REGISTRY_URL} ] && [ ! -z ${INTERNAL_REGISTRY_USERNAME} ] && [ ! -z ${INTERNAL_REGISTRY_PASSWORD} ] ; then
```

Comment:
unfortunately the variables you check here are never created, we need to parse them out of the project variables out first, see the code in kubectl-build-deploy-dind

https://github.com/amazeeio/lagoon/blob/a9807db86b9fbe0826a7211f2322303ad67bf362/images/kubectl-build-deploy-dind/build-deploy.sh#L11-L15

Reply:
Cool - changed!